rusty-machine/src/learning/gp.rs

```diff
@@ -184,24 +184,24 @@ impl<T: Kernel, U: MeanFunc> GaussianProcess<T, U> {
     /// Requires the model to be trained first.
     ///
     /// Outputs the posterior mean and covariance matrix.
-    pub fn get_posterior(&self, data: &Matrix<f64>) -> (Vector<f64>, Matrix<f64>) {
+    pub fn get_posterior(&self, inputs: &Matrix<f64>) -> (Vector<f64>, Matrix<f64>) {
         if let (&Some(ref t_mat), &Some(ref alpha), &Some(ref t_data)) = (&self.train_mat,
                                                                           &self.alpha,
                                                                           &self.train_data) {
-            let mean = self.mean.func(data.clone());
+            let mean = self.mean.func(inputs.clone());
 
-            let post_mean = mean + self.ker_mat(data, t_data) * alpha;
+            let post_mean = mean + self.ker_mat(inputs, t_data) * alpha;
 
-            let test_mat = self.ker_mat(data, t_data);
-            let mut v_data = Vec::with_capacity(data.rows() * data.cols());
+            let test_mat = self.ker_mat(inputs, t_data);
+            let mut v_inputs = Vec::with_capacity(inputs.rows() * inputs.cols());
```

Comment:
This shouldn't be `v_inputs`. By `v_data` I meant `variance_data`, i.e. the underlying data for the variance matrix.